module/platform/darwin/SensorFilter/src/SensorFilter.cpp

```diff
@@ -107,11 +108,14 @@ namespace platform {
             , load_sensor()
             , footlanding_rFWw()
             , footlanding_Rfw()
-            , footlanding_Rwf() {
+            , footlanding_Rwf()
+            , theta(arma::fill::zeros) {
```

Comment:
Remove this, there should always be a single source of truth for any value

Reply:
Will perform additional testing to see if gyro conversion fixed reading angular positions from the world transform. If not, this will remain as-is until the Euler angles can be found using the world transform